cypress/local-only/tests/integration/fileAPetitionUpdated/file-a-petition-step-1-business--partnership-tax-matters-partner.cy.ts

```diff
@@ -0,0 +1,371 @@
+import { InputFillType, selectInput, textInput } from './petition-helper';
+import { loginAsPetitioner } from '../../../../helpers/authentication/login-as-helpers';
+
```

Comment:
LOVE all the cypress tests you all have written. AWESOME.

Reply:
I agree! Although I wonder if we should add accessibility tests for the new screens in a follow-up PR.